libmscore/score.cpp

```diff
@@ -4217,6 +4217,20 @@ int Score::duration()
       return lrint(utick2utime(rs->utick + rs->len()));
       }
 
+//---------------------------------------------------------
+//   durationWithoutRepeats
+//---------------------------------------------------------
+
+int Score::durationWithoutRepeats()
+      {
+      masterScore()->setExpandRepeats(false);
```

Comment:
Is it not necessary to turn this back on when done?  I have a hunch this is responsible for https://musescore.org/en/node/308698